tools/calling_intervals_yaml_to_beds.cwl

```diff
@@ -0,0 +1,34 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: calling_intervals_yaml_to_beds
+doc: "Convert Calling Intervals YAML to interval BEDs"
+requirements:
+  - class: InlineJavascriptRequirement
+  - class: ShellCommandRequirement
+  - class: DockerRequirement
+    dockerPull: 'dmiller15/python-util:3.9'
```

Comment:
Is this a public docker ?

Reply:
It is public, yes. I will be moving this over to cavatica when I have the chance.